docs/ref/models/querysets.txt

```diff
@@ -2474,6 +2474,51 @@ each model instance (if the database normally supports it).
     In older versions, enabling the ``update_conflicts`` parameter prevented
     setting the primary key on each model instance.
 
+.. versionadded:: 5.1
+
+    The ``where_clause`` parameter was added.
```

Comment:
This annotation should be moved to the end of the section for the `bulk_create()` method.